DependencyInjection/GrossumStaticPageExtension.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace Grossum\StaticPageBundle\DependencyInjection;
+
+use Symfony\Component\Config\FileLocator;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Loader;
+use Symfony\Component\DependencyInjection\Exception\LogicException;
+use Symfony\Component\HttpKernel\DependencyInjection\Extension;
+
+use Sonata\EasyExtendsBundle\Mapper\DoctrineCollector;
+
+class GrossumStaticPageExtension extends Extension
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function load(array $configs, ContainerBuilder $container)
+    {
+        $configuration = new Configuration();
+        $config        = $this->processConfiguration($configuration, $configs);
+
+        $registeredBundles = $container->getParameter('kernel.bundles');
+
+        if (!isset($registeredBundles['GrossumCoreBundle'])) {
```

Comment:
I think, we can use array for required bundles.
Like a:
```
protected $requiredBundles = ['GrossumCoreBundle', 'IvoryCKEditorBundle'];
```

and here something like:

```
foreach ($this->requiredBundles as $bundle) {
    if (!isset($registeredBundles[$bundle]))) {
        throw new LogicException(sprintf('GrossumStaticPageBundle required %s', $bundle));
    }
}
```